nodejs/eks/nodegroup.ts

```diff
@@ -150,12 +150,12 @@ export interface NodeGroupBaseOptions {
     /**
      * The tags to apply to the NodeGroup's AutoScalingGroup.
      */
-    autoScalingGroupTags?: { [key: string]: string };
+    autoScalingGroupTags?: pulumi.Input<{ [key: string]: string }>;
 
     /**
      * The tags to apply to the CloudFormation Stack of the Worker NodeGroup.
      */
-    cloudFormationTags?: { [key: string]: string };
+    cloudFormationTags?: pulumi.Input<{ [key: string]: string }>;
```

Comment:
I suspect (but am not sure) that the usage of ‘cloudFormationTags` needs to change to support this new type. 

At least the usage as written currently is highly misleading, as it does an untyped cast into a type that is no longer guaranteed to be correct. 

It’s possible that it still just works because of how things get serialized before passing to the resource provider - but it might make sense to fix up the types to be correct as well.

Reply:
FWIW - I expect this is true for several of the other tag properties that have changed in this PR.

Reply:
Resolved using Pat's suggestion below